src/classes/keyboard.class.js

```diff
@@ -343,9 +344,8 @@ class Keyboard {
                 }, 100);
             }
 
-            if (e.key === "Enter") {
+            if(this.container.dataset.passwordMode == "false")
```

Comment:
Shouldn't this be
```suggestion
            if(this.container.dataset.passwordMode == "false" && e.key === "Enter")
```
?

Reply:
Yes, my bad